src/be_db_block.erl

```diff
@@ -142,8 +142,11 @@ maybe_write_snapshot(Height, SnapshotHash, SnapshotDir, Chain) ->
             {error, not_found} -> blockchain:get_snapshot(Height, Chain);
             Other -> Other
         end,
+    LatestIOList = io_lib:format("~p~n~p~n", [Height, base58:binary_to_base58(SnapshotHash)]),
```

Comment:
I think it would be a Good Idea to make this self documenting with key=value or json you know, for the day when we rewrite everything in rust.